Tasks/SqlAzureDacpacDeploymentV1/README.md

```diff
@@ -71,9 +74,11 @@ The parameters of the task are described in details, including examples, to show
 
 - **Service Principal** : Uses the Authentication data from **Azure Subscription**. 
 
+- **Windows Authentication** : Uses the credentials of the self-hosted pipeline agent.
+
 **SQL DB Details**
 
-- **Azure SQL Server Name\*:** The connection string for the Azure SQL Server and the format is same as that is followed in SQL Server Management Studio. For example, FabrikamSQL.database.windows.net, 1433 or FabrikamSQL.database.windows.net are both valid Azure SQL Server names.
+- **SQL Server Name\*:** The connection string for the Azure SQL Server and the format is same as that is followed in SQL Server Management Studio. For example, FabrikamSQL.database.windows.net, 1433 or FabrikamSQL.database.windows.net are both valid Azure SQL Server names.
```

Comment:
since this is an area of confusion, maybe we document it as "SQL Server or Azure SQL Server name"

Reply:
The 'or' makes sense; I've updated the readme to include those changes.

Other sections have been updated to better reflect what's available in Azure in 2024. However, parts of this file, such as Visual Studio versions and some of the verbiage of Azure resources, are still outdated and need significant TLC.  

Is there some other documentation around SQLPackage and IDEs to plug in here?